src/org/parosproxy/paros/view/AbstractParamDialog.java

```diff
@@ -238,21 +247,39 @@ private AbstractParamContainerPanel getJSplitPane() {
 
 
     /**
-     * If multiple name use the same panel
+     * Adds the given panel with the given name positioned under the given parents (or root node if none given).
+     * <p>
+     * If not sorted the panel is appended to existing panels.
      *
-     * @param parentParams
-     * @param name
-     * @param panel
+     * @param parentParams the name of the parent nodes of the panel, might be {@code null}.
+     * @param name the name of the panel, must not be {@code null}.
+     * @param panel the panel, must not be {@code null}.
+     * @param sort {@code true} if the panel should be added in alphabetic order, {@code false} otherwise
      */
     // ZAP: Added sort option
```

Comment:
Could be removed? Sort is handled numerous places here and not documented in this way....